modules/survey_accounts/test/survey_accountsTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Survey accounts automated integration tests
+ *
+ * PHP Version 5
+ *
+ * @category Test
+ * @package  Loris
+ * @author   Ted Strauss <ted.strauss@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+
+require_once __DIR__ . "/../../../test/integrationtests/LorisIntegrationTest.class.inc";
+class survey_accountsTestIntegrationTest extends LorisIntegrationTest
+{
+    protected $url = 'http://localhost/main.php?test_name=survey_accounts';
+
+    /**
+     * Tests that, when loading the Survey accounts module, some
+     * text appears in the body.
+     *
+     * @return void
+     */
+    function testSurveyAccountsDoespageLoad()
```

Comment:
Can you also add a test for the subpage loading:

?test_name=survey_accounts&subtest=add_survey